AlphaWallet/Core/Types/WalletBalance.swift

```diff
@@ -28,7 +28,7 @@ struct WalletBalance: Equatable {
     }
 
     var etherAmountShort: String? {
-        guard let token = etherTokenObject, let value = token.valueDecimal else { return nil }
```

Comment:
Could you help to separate these changes out? Looks like they aren't related to image sizes?